bin/pgenv

```diff
@@ -229,14 +334,24 @@ case $1 in
         cd src
 
         # Download the source if wee don't already have it.
-        PG_TARBALL="postgresql-$v.tar.bz2"
+        # WARNING: older PostgreSQL used .tar.gz instead of .tar.bz2,
+        # so if the version is behind 8 use the first format, otherwise
+        # try to get the most compressed archive
+        if [[ $v =~ ^[1-7]\. ]]; then
```

Comment:
Have you tested building these earlier versions? I stopped with 8.0. Earlier versions are likely to require patches, I think.

Reply:
7.4 can be built (without contrib) on freeBSD 11, I haven't tried something older.